scripts/reformat-all-commands.sh

```diff
@@ -93,8 +94,20 @@ for ((i=${#sorted_files[@]}-1; i>=0; i--)); do
         sed -i.bak "s/\(\[[[:alnum:][:space:]]*\]\)(jsonspec\.md)/\1(..\/jsonspec.md)/g" "../docs/get-details/dapps/avm/teal/opcodes/v${version}.md"
         echo "  - v${version}.md" >> $pages_file
 
-        if [ $i -eq $(( ${#opcode_files[@]} - 1 )) ]; then
-          echo "- [v${version} - Current Version](v${version}.md)" >> "$index_file"
+        if [ $i -eq $(( $vfuture_version - 1)) ]; then
+            # The most efficient POSIX-compliant way to insert a line that I could find.
```

Comment:
You could use `sed` to insert something right under the `title:` line. Something like this works for me locally (though you may need to adjust the newlines to work in the script):

```bash
sed -i.bak '/^title:/a\
\
!!! Warning "This page contains the opcodes currently available in vFuture (not on Mainnet) and may change before release."\
' "../docs/get-details/dapps/avm/teal/opcodes/v${version}.md"
```